config/config.go

```diff
@@ -912,6 +912,9 @@ type StateSyncConfig struct {
 
 	// Timeout before considering light block verification failed
 	VerifyLightBlockTimeout time.Duration `mapstructure:"verify-light-block-timeout"`
+
+	// Time before which a blacklisted witness can not be added back as a provider
+	BlacklistTTL time.Duration `mapstructure:"verify-light-block-timeout"`
```

Comment:
this mapstructure name maybe should be `blacklist-ttl` ?

Reply:
This was just pushed, can you pull latest? Should be there. Thanks